x/bounty/types/genesis.go

```diff
@@ -27,13 +29,12 @@ func ValidateGenesis(data *GenesisState) error {
 	programIds := make(map[uint64]bool)
 	for _, program := range data.Programs {
 		if program.ProgramId > data.StartingProgramId {
-			return fmt.Errorf("error program id")
+			return ErrProgramID
 		}
 
 		_, ok := programIds[program.ProgramId]
 		if ok {
-			//repeat program
-			return fmt.Errorf("repeat programId:%d", program.ProgramId)
+			return sdkerrors.Wrapf(ErrProgramID, "repeat programId:%d", program.ProgramId)
```

Comment:
`duplicate programID: %d`
and it should print out the programs themselves to be compared